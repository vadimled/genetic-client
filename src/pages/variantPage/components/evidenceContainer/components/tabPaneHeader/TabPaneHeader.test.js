import "jest-dom/extend-expect";

describe("TabPaneHeader", () => {
  it("snapshot", () => {
    expect(true).toBeTruthy();
  });
});
