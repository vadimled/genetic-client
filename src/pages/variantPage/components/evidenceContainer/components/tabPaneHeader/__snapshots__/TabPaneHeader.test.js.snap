// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabPaneHeader /> snapshot 1`] = `
<DocumentFragment>
  <div
    class="tab-pane-content-wrapper"
  >
    <div
      class="flex flex-row justify-start content-center"
    >
      <div
        class="title"
      >
        Publications
      </div>
      <div
        class="amount"
      >
        (5)
      </div>
    </div>
  </div>
</DocumentFragment>
`;
